internal/upf/ebpf/xdp/utils/qer.h

```diff
@@ -31,10 +31,10 @@ struct qer_info {
 	__u8 ul_gate_status;
 	__u8 dl_gate_status;
 	__u8 qfi;
-	__u32 ul_maximum_bitrate;
-	__u32 dl_maximum_bitrate;
-	__u64 ul_start;
-	__u64 dl_start;
+	__u64 ul_maximum_bitrate;
+	__u64 dl_maximum_bitrate;
+	volatile __u64 ul_start;
+	volatile __u64 dl_start;
```

Comment:
I was not aware of this behavior, could we have the same types of problems in other parts of the XDP program?